src/utils/storage/local.ts

```diff
@@ -0,0 +1,20 @@
+const PREFIX = 'SAFE__'
+
+const prefixKey = (key: string): string => `${PREFIX}${key}`
```

Comment:
We already have a function for getting the prefix?

Reply:
Can you point me to it?

Reply:
`getStoragePrefix()` at `src/utils/storage/index.ts`

Reply:
That one is for immortal.

Reply:
Do we want to follow a versioning convention though, just in case we make any drastic changes?

Reply:
We'll have to live with two keys for now: the legacy immortal ones and the new `SAFE` ones.
The first ones, we'll need to eventually migrate (not a high prio I think).

As for versioning, We can always slap a version on top of the new key, e.g. `SAFE_v2`. Once there's a need.